testament/specs.nim

```diff
@@ -317,6 +323,7 @@ proc initSpec*(filename: string): TSpec =
   result.file = filename
 
 proc isCurrentBatch*(testamentData: TestamentData; filename: string): bool =
+  # TODO: move elsewhere
```

Comment:
It would be nice to elaborate on the reasons why things should be moved elsewhere. As personal reminders (in private projects) TODO's like this are fine, but in community projects they are very confusing when encountered by people who did not write them.

Reply:
Yep, that was an oversight on my side. I have adjusted the comment.